torch/_inductor/pattern_matcher.py

```diff
@@ -1514,9 +1515,9 @@ def get_file_template() -> str:
         formatted_imports = f"from torch._inductor.pattern_matcher import (\n   {formatted_imports},\n)\n"
         return f"{file_template}{formatted_imports}"
 
-    if not SERIALIZED_PATTERN_PATH.is_dir():
+    if not TORCHINDUCTOR_SERIALIZED_PATTERN_PATH.is_dir():
```

Comment:
Please revert renaming this variable. When I asked to rename the envvar I was just talking about the external-facing string (as you did above), not the internal name. We're already in inductor - no need to churn the codebase unnecessarily.